src/mixins/AddSendRequestByMessage.ts

```diff
@@ -184,9 +184,9 @@ export function AddSendRequestByMessage<TBase extends ConnectorTUIBaseConstructo
           type: "select",
           name: "owner",
           choices: [
-            { title: "You (Connector)", value: "" },
-            { title: "The other Side", value: "recipient" },
-            { title: "Third Party", value: "thirdParty" },
+            { title: "The other side or the third party", value: "" },
```

Comment:
I don't want to annoy you, BUT it should be "a third party" instead of "the third party" because multiple third parties are possible, if `""` or `"thirdParty"` is specified for the `owner` of the ThirdPartyRelationshipAttributeQuery :D

Same below.

Reply:
true